tests/e2e/common-operator-integ-suite.sh

```diff
@@ -191,18 +214,50 @@ if [ "${SKIP_BUILD}" == "false" ]; then
     get_internal_registry
   fi
 
-  # BUILD AND PUSH IMAGE
+  # BUILD AND PUSH OPERATOR IMAGE
   build_and_push_image
 
-  if [ "${OCP}" == "true" ]; then
-    # This is a workaround
-    # To avoid errors of certificates meanwhile we are pulling the operator image from the internal registry
-    # We need to set image $HUB to a fixed known value after the push
-    # This value always will be equal to the svc url of the internal registry
-    HUB="image-registry.openshift-image-registry.svc:5000/sail-operator"
+  # If OLM is enabled, deploy the operator using OLM
+  if [ "${OLM}" == "true" ]; then
+    # Set the platform to Kubernetes by default.
+    # We are skipping the deploy via OLM test on OCP because the workaround to avoid the certificate issue is not working.
```

Comment:
If we have an issue/Jira to track it, we can include it here.

Reply:
Added the comment also